src/data.js

```diff
@@ -40,6 +40,28 @@ module.exports = [
       'TailwindCSS',
     ],
   },
+  {
+    name: 'Dennis Mathenge',
+    description: 'Web Developer',
+    url: 'https://creativehubspace.com/uses',
+    twitter: '_dennisnm',
```

Comment:
```suggestion
    twitter: '@_dennisnm',
```
this is probably needed to work

Reply:
Twitter handle updated.